x-pack/plugins/cases/common/api/cases/case.ts

```diff
@@ -164,6 +164,43 @@ export const CasePostRequestRt = rt.intersection([
   }),
 ]);
 
+const CasesFindRequestSearchFieldsRt = rt.union([
```

Comment:
I think we should remove fields that are dates or numbers and add the tags and the assignee uid. Specifically:

Add: `tags`, `assignees.uid`
Remove: `duration`, `closed_at`, `created_at`, `updated_at`, `settings.syncAlerts`

What do you think?

Reply:
cc @jonathan-buttner

Reply:
To be honest I was not aware you can search for exact booleans, dates, and numbers. I forgot that we discussed with @adcoelho that we will omit from the list any field that can be searched with other fields. For example, the `tags` 🙂.

Reply:
> About the other fields, as you said it does not make sense to support that with the `searchFields`. What do you all think?

I'm pretty sure we had discussed the specific timestamp thing 😅 I'll remove them, it isn't very usable.

Reply:
>I'm pretty sure we had discussed the specific timestamp thing 😅 I'll remove them, it isn't very usable.
You are right. Sometimes you see the code, you realise you were wrong and it does not make sense anymore 🙂.